assets/css/NewEvaluation.less

```diff
@@ -0,0 +1,123 @@
+body {
+    box-sizing: border-box;
+    background-color: #f7f7f7;
+    
+}
+
+.dataForm {
+    width: 100%;
+    text-align: left;
+    padding: 15px;
+    margin-bottom: 25px;
+    
+     input{
+         text-align: left;
+         width: 32%;
+         min-height: 25px;
+         text-decoration: underline;        
+    }
+}
+
+.table{
+    width: 100%;
+    height: 5rem;
+    border-collapse: collapse;
+    text-align: center;
+
+    .row1{
+
+        background-color: #03a9f4;
+        color: white;
+    }
+
+    .row2{
+        border: solid thin black;
+
+        input{
+            background-color: white;
+        }
+    }
+}
+
+.textA {
+    resize: none;
+    width:100%;
+    background-color: #f7f7f7;
+}
+
+.theForm {
+    margin: 0 20% 0 20%;
+    width: 66.66%;
+    display: block;
+}
+
+fieldset {
+    border: solid thin black;
+    margin: 25px 0 25px 0;
+
+    legend{
+        font-weight: bold;
+    }
+}
+
+.evalForms {
+    list-style: none;
+
+
+    li {
+        float: left;
+        width:16.6%;
+        min-height: 36px;
+    }
+}
+
+.evalForms-title {
+    color: #03a9f4;
+    font-weight: bold;
+    font-size: 130%;
+    padding: 10px 25px 10px 25px ;
+}
+
+.descrip {
```

Comment:
When coding, it is best to avoid shorthands whenever possible, unless they're common enough. That is so other people working on your code can easily understand what's going on, even if they're unfamiliar with the particular shorthand / contraction / abbreviation etc.